browse/templates/home/home.html

```diff
@@ -28,11 +28,20 @@
 </p>
 {#- TODO: define display order in taxonomy? -#}
 {{- group_section(('grp_physics','grp_math','grp_cs','grp_q-bio','grp_q-fin','grp_stat','grp_eess','grp_econ')) }}
-{% endblock content -%}
 
-{%- block footer_content -%}
-{%- include "long_trailer.html" -%}
-{%- endblock footer_content -%}
+<hr />
+{#- TODO: remove because of new footer? -#}
+<h2>About arXiv</h2>
```

Comment:
I forgot about this section of the homepage originally. I wonder if it should be removed or revised in light of new footer.

Reply:
I'd leave it for now. Some other time there will be a larger scale redesign and it can get done during that.